packages/genesys-spark-components/src/components/stable/gux-time-picker/example.html

```diff
@@ -45,6 +45,27 @@ <h3>Disabled</h3>
 
   <h3>Has Error</h3>
   <gux-time-picker has-error></gux-time-picker>
+
+  <h3>Min of "03:15" and Max of "10:45" with 24 Hour Time Format</h3>
```

Comment:
I would maybe expect that min and max are inclusive of the min and max values. For example, I would think that 3:15 and 10:45 should be included in the list?
![image](https://github.com/user-attachments/assets/b6f3a269-1236-4b49-98e6-c67418e379d7)

Reply:
That's a good catch. I'll make a fix for this.

Reply:
fixed